Library/QuickPulseStateManager.ts

```diff
@@ -29,6 +29,8 @@ class QuickPulseStateManager {
     private _metrics: {[name: string]: Contracts.MetricQuickPulse} = {};
     private _documents: Contracts.DocumentQuickPulse[] = [];
     private _collectors: {enable: (enable: boolean) => void}[] = [];
+    private _rediectedHost: string = null;
```

Comment:
Just curious - why this vs updating the host in config? Is the idea that we don't want to expose the redirection when it happens?

Reply:
We do not want to make this change persistent, if the agent is restarted, then it should start polling the endpoint it was originally configured to. It will be redirected again